docs/api.rst

```diff
@@ -6,6 +6,11 @@ This page lists all of the interfaces exposed by the `treq` package.
 Making Requests
 ---------------
 
+The :py:mod:`treq` module provides several shortcut functions for making
```

Comment:
Would "convenience" be a better word here (and below) than "shortcut"?

Reply:
Sure, that makes sense to me.